docs/docs-content/deployment-modes/agent-mode/install-agent-host.md

```diff
@@ -93,8 +93,8 @@ Palette. You will then create a cluster profile and use the registered host to d
   :::
 
   - If installing the FIPS version of Agent Mode on a Rocky Linux edge host, you must configure your SELinux policies to
-    grant rsync the required host permissions. Follow the process below to apply the necessary configurations before
-    installing Agent Mode.
+    grant rsync the required host permissions. If you are using Cilium, you must also configure the appropriate
```

Comment:
The firewalld rules are needed if firewalld is enabled and your desired CNI is Cilium.